rust_snuba/src/processors/snapshots/rust_snuba__processors__tests__schemas@snuba-queries-QuerylogProcessor-snuba-queries__1__snuba-queries-empty-trace-id.json.snap

```diff
@@ -0,0 +1,102 @@
+---
+source: src/processors/mod.rs
```

Comment:
what's the reason for adding this snapshot for the querylog consumer?

Reply:
it was needed to make some tests pass -- i think because the schemas library was updatedd